ts/components/messages/MessageAgenda.tsx

```diff
@@ -159,19 +166,27 @@ type OwnProps = {
   onLongPressItem: (id: string) => void;
   onMoreDataRequest: () => void;
   selectedMessageIds: Option<Set<string>>;
+  lastDeadlineId: Option<string>;
 };
 
 type Props = OwnProps & SelectedSectionListProps;
 
 type State = {
   itemLayouts: ReadonlyArray<ItemLayout>;
   prevSections?: Sections;
+  firstSectionListUpdate: boolean;
```

Comment:
Since _firstSectionListUpdate_ and _elementVisible_ are not used they should be removed